src/hooks/useNavigation.js

```diff
@@ -7,15 +7,26 @@ export const useNavigation = (origin, containerRef, axis, elementsSelector = '[d
   const getAllElements = () => document.querySelectorAll(elementsSelector)
 
   const getSelectedElement = () => {
-    return document.querySelector('[nav-selected=true]')
+    const allNavSelectedTrue = document.querySelectorAll('[nav-selected=true]')
+    const currentSelector = elementsSelector.slice(1, -1)
+    let selectedElement
+    allNavSelectedTrue.forEach(element => {
+      if (element.getAttribute(currentSelector)) {
+        selectedElement = element
```

Comment:
I'm assuming there's always a `nav-selected=true` element with the corresponding `currentSelector`, but let's talk if that's not a safe assumption